src/BaGet.Aliyun/Configuration/AliyunOSSOptions.cs

```diff
@@ -0,0 +1,22 @@
+using System.ComponentModel.DataAnnotations;
+using BaGet.Core;
+
+namespace BaGet.Aliyun.Configuration
+{
+    public class AliyunOSSOptions
```

Comment:
What do you think of the name `AliyunStorageOptions`?

```suggestion
    public class AliyunStorageOptions
```